swerve_commander:
  ros__parameters:
    wheelbase: 0.705
    wheel_track: 0.30
    wheel_radius: 0.12
swerve_odometer:
  ros__parameters:
    wheelbase: 0.705
    wheel_track: 0.30
    wheel_radius: 0.12
